src/main/java/org/apache/sling/bundleresource/impl/PathMapping.java

```diff
@@ -44,18 +46,20 @@ public static PathMapping[] getRoots(final String rootList) {
             final String resourceRoot = entry.getValue();
             final String pathDirective = entry.getDirectiveValue(DIR_PATH);
             final String expandDirective = entry.getDirectiveValue(DIR_JSON);
+            final boolean skipSettingResourceTypeProperty = Boolean.parseBoolean(entry.getDirectiveValue(SKIP_SETTING_RESOURCE_TYPE_FLAG));
```

Comment:
Maybe "hideResourceTypeProperties" is the better name (and an according constant name value).

Reply:
It was proposed by @cziegeler in the [jira comment](https://issues.apache.org/jira/browse/SLING-11504?focusedCommentId=17574690&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-17574690)

I think that skipSettingResourceTypeProperty describes the actual flag behavior. Hiding a property may be understood in different ways.